server/prometheus.server.ts

```diff
@@ -0,0 +1,64 @@
+import * as client from 'prom-client';
+
+// Recreate the singleton function here, we don't want to import it from
+// the utils because this is used by server index.ts which can only import the JS
+// compiled directly from this file, and I don't want to move it to the server folder
+function singleton<Value>(name: string, value: () => Value): Value {
+    const g = global as any;
+    g.__singletons ??= {};
+    g.__singletons[name] ??= value();
+    return g.__singletons[name];
+}
```

Comment:
We can now use [`@epic-web/remember`](https://www.npmjs.com/package/@epic-web/remember) :)

Reply:
replaced with remember.